api/src/main/java/io/cloudevents/rw/CloudEventRWException.java

```diff
@@ -56,6 +56,10 @@
          * Error while converting CloudEventData.
          */
         DATA_CONVERSION,
+        /**
+         * Invalid content type or spec version
+         */
+        PARSE_EXCEPTION,
```

Comment:
I think this should be named `UNKNOWN_ENCODING`, `PARSE_EXCEPTION` is too generic.

Reply:
OK. Sure

Reply:
I have updated everything requested but the build is still failing for some reason. Please look into it.